plugins/ssr/ssrfwd.ml

```diff
@@ -361,16 +348,14 @@ let intro_lock ipats =
              Proofview.tclDISPATCH
                (ncons (ng - 1) ssrsmovetac @ [Proofview.tclUNIT ()]) in
   let protect_subgoal env sigma hd args =
+    Ssrcommon.tacMK_SSR_CONST "Under_rel" >>= fun under_rel ->
```

Comment:
Note that in case this is turned into a universe polymorphic constant, this kind of move is changing the semantics. For now it should be fine though.